src/xtgeoapp_grd3dmaps/aggregate/_grid_aggregation.py

```diff
@@ -297,7 +300,8 @@ def _property_to_map(
     if weights is not None:
         assert method in [AggregationMethod.MEAN, AggregationMethod.SUM]
     data = prop[0][cols] if len(prop) == 1 else prop[cols]
-    # Small hack due to a slight difference between calculating mass and other properties
+    # Small hack due to a small difference between calculating mass and other properties
+    # pylint: disable=fixme
     # TODO: Implement a better solution
```

Comment:
Could also remove the TODO since we have an issue for it, maybe copy the line data = ... to remember where it was. Then we don't need to disable the pylint message.